workflow.md

```diff
@@ -0,0 +1,73 @@
+# InnerSource Patterns Workflow
+
+This document describes the workflow for writing, reviewing and publishing
+InnerSource ideas, donuts and patterns. It utilizes Pull Requests to facilitate
+collaborative reviews.
+
+There are two separate repositories needed for this workflow:
+
+* [InnerSourcePatterns][patternsRepo]
+* [InnerSourceCommons][commonsRepo]
+
+### InnerSourcePatterns repository
+
+This is a private repository where new ideas, donuts and patterns are published
+and reviewed by the patterns community **prior to** publishing them on
+[innersourcecommons.org][commons] ,Inside of this
+repository we're using the standard GitHub workflow where we have one main
+branch - the master branch.  Contributions are done via fork and pull-requests.
+
+### InnerSourceCommons repository
+
+This is where ideas, donuts and patterns will be published **after** they have
+been reviewed and accepted by the reviewers. Inside of this repository we're
+using the standard GitHub workflow where we have one main branch - the master
+branch. Contributions are done via fork and pull-requests. This repository
+contains the sources for the GitHub pages website for innersourcecommons.org.
+
+## Workflow
+
+### Working inside of the InnerSourcePatterns repository
+
+# [Create an issue] for each new idea, donut or pattern in the 
+  [patterns repository][patternsRepo]. The issue should
+  * contain the name of the pattern,
+  * contain a short description (at least the problem) and
+  * be labeled with the appropriate label (_idea_, _donut_, _pattern_) 
+# Create a new branch either in your clone or fork of the 
+  [patterns repository][patternsRepo]. Please use the following pattern for
+  naming branches: `pattern/<patternName>`. Example: 
+  `pattern/contractedContributor`.
+# Create a _Markdown_ file with the description of the _idea_, _donut_ or
+  _pattern_ and store it either the `ideas`, `donuts` or `patterns` 
```

Comment:
added the subdirectories as a proposal. I think it would make sense to do this in order to avoid having a user of the repo guess whether a file describes an _idea_, a _donut_ or a _pattern_.